specs/pvs/corla.pvs

```diff
@@ -486,11 +527,15 @@ END ballot_manifests
 % @relation well_formed_vote inherit vote
```

Comment:
In the C.R.S. the adjective "verified" is used, thus there is a notion of "verified" ballot manifest and "unverified" ballot manifest.  Anytime you have an adjective or adverb modifier that encodes a decidable truth-hood (i.e., being verified is either true or false of any ballot manifest in CO), then we encode that with a predicate subtype, `pred[T]`.  This is shorthand notion for a function from `T` to `bool`, `[T -> bool]`.